glean-core/python/setup.py

```diff
@@ -35,11 +35,11 @@
 setup_requirements = []
 
 if sys.platform == "linux":
-    shared_object_extension = "so"
+    shared_object = "libglean_ffi.so"
 elif sys.platform == "darwin":
-    shared_object_extension = "dylib"
+    shared_object = "libglean_ffi.dylib"
 elif sys.platform.startswith("win"):
-    shared_object_extension = "dll"
+    shared_object = "glean_ffi.dll"
```

Comment:
*invents a new operating system with completely different names just to mess with you*